### DISCUSSION TITLE
Newest added repositories should appear at the top of the list

### DISCUSSION THREAD

**Discussion by stuymedova** [community/community#17118] [Other Feature Feedback, Questions, & Ideas]
I think it would be nice if repositories added latest would appear at the top of the list, as stars do. They tend to be most relevant and interesting to me, so I'd like to access them more quickly. Currently they get added to the bottom.

Edit: Alternatively, perhaps could be maintained the order in which the repositories were starred.

**Comment by Tuanhuu773** [comment]
> I think it would be nice if repositories added latest would appear at the top of the list, as stars do. They tend to be most relevant and interesting to me, so I'd like to access them more quickly. Currently they get added to the bottom.
> 
> Edit: Alternatively, perhaps could be maintained the order in which the repositories were starred.

**Reply by Tuanhuu773** [reply]
0898361734

**Reply by Tuanhuu773** [reply]
> 0898361734

Ok

**Comment by bullardjonfa487** [comment]
Listserv

**Comment by Joookeryyy** [comment]
VK: music, video,
messenger

**Reply by bullardjonfa487** [reply]
Call the police

**Comment by jfullstackdev** [comment]
would the `Newest` filter not be enough ? 

![image](https://user-images.githubusercontent.com/47092464/198875533-cee22589-17d6-4760-b23c-fffccef57b86.png)
